update-lb4-docs.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs-extra');
+const path = require('path');
+
+const srcDocs = path.resolve(__dirname,'node_modules/@loopback/docs/site');
+const srcImages = path.resolve(srcDocs, 'imgs');
+const destDocs = path.resolve(__dirname, 'pages/en/lb4');
+const destImages = path.resolve(__dirname, 'images/lb4');
```

Comment:
Now we copy markdown and image files to two different places. Is it going to be  a problem for image links such as `./imgs/my-diagram.png`?

Reply:
That's a good point. I've removed `images/lb4` entirely, and the images will live in `pages/en/lb4/imgs`. That way we don't have to worry about image links too (I've verified it).